cpp/include/cuspatial/soa_readers.hpp

```diff
@@ -86,8 +89,8 @@ read_xy_points_soa(const char *filename,
  *          column(2): FLOAT64 x-coordinates of concatenated polygons.
  *          column(3): FLOAT64 y-coordinates of concatenated polygons.
 **/
-std::vector<cudf::column>
-read_polygons(std::string filepath, 
+std::vector<std::unique_ptr<cudf::column>>
+read_polygon_soa(const char *filename,
```

Comment:
Could this be a `std::string const&` instead of a `char const*`?

Reply:
That would be a very un-legacylike improvement, will do.